srcpkgs/genie/patches/no-git.diff

```diff
@@ -0,0 +1,23 @@
+diff --git a/scripts/release.lua b/scripts/release.lua
```

Comment:
suggestion: write `version.h` directly from the template and simply drop its handling in `release.lua`, might make it less fragile

Reply:
Reasonable suggestion @Piraty, it is done.